renovate.json5

```diff
@@ -36,6 +36,12 @@
             "matchDatasources": ["maven"],
             "registryUrls": ["https://repo1.maven.org/maven2/", "https://plugins.gradle.org/m2/", "https://dl.google.com/android/maven2/", "https://jitpack.io"]
         },
+        {
+            "packagePatterns": [
+                "^com.google.guava:"
+            ],
+            "versionScheme": "docker"
```

Comment:
If it's working is there any way to find the underlying versionScheme regex and tweak it?

It's weird to see docker here. We could come with a regex that ignore what is after - ?

Reply:
I'm not that familiar with Renovate, just searched for the issue and this seems to be a solution that is also used by quite a lot of other repositories. If you can do something with a regex, feel free to do so.

Reply:
Based on another search it is renamed to `versioning` (will push a change for that) and a list of values, not a regex: https://docs.renovatebot.com/configuration-options/#versioning

Reply:
Actually I think the way to fix what we want is with the parameter above in the documentation `versionCompatibility`

something like this should do the trick

```json
            "versionCompatibility": "^(?<version>.*)-android$"
            "versioning": "semver"
```

I'll push the change and check if it works today